docs/welcome/installation.md

```diff
@@ -26,11 +26,132 @@ Before installing, ensure your site meets these requirements:
 2. Extract the plugin files
 3. Upload the plugin folder to `/wp-content/plugins/`
 4. Activate through the WordPress admin interface
+---
 
-## Verification
+### Composer Installation
+
+To install and manage Secure Custom Fields in your WordPress theme or plugin, it is recommended to use Composer.
+
+This ensures that SCF is properly versioned, loaded automatically, and easy to update.
+
+### Why integrate **Secure Custom Fields (SCF)** with Composer?
+
+Integrating SCF using Composer offers several important advantages for the professional development of WordPress plugins and themes:
+
+- **Centralized dependency management:**  
+  Composer allows you to declare, install, and update SCF easily along with other libraries needed in your project.
+
+- **Version control:**  
+  You can lock a specific version of SCF in your `composer.json`, ensuring that all development and production environments use exactly the same version, avoiding unexpected errors.
+
+- **Simplified deployment and automation (CI/CD):**  
+  Composer installs SCF automatically during deployment processes (`composer install`), eliminating the need to manually copy files.
+
+- **Automatic autoloading:**  
+  Composer handles PHP class autoloading. Integrating SCF this way makes your code cleaner, safer, and PSR-4 compliant.
+
+- **Reduced repository size:**  
+  By installing SCF as an external dependency, you avoid having redundant copies of the plugin in your project, keeping your repository lighter and easier to maintain.
+
+- **License compliance:**  
+  Composer records the licenses of all dependencies used, which is very useful if you distribute your plugins or themes and need to meet legal or auditing requirements.
+
+- **Facilitates project changes:**  
+  By managing SCF as a dependency, any theme or plugin can change its structure or installation system without affecting its functionality, ensuring greater flexibility and compatibility in development.
+
+---
+
+### How to Load and Use **Secure Custom Fields (SCF)** with Composer
+
+Add the following configuration to your `composer.json` file:
+
+```json
+{
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org"
+    }
+  ],
+  "extra": {
+    "installer-paths": {
+      "vendor/{$name}/": ["type:wordpress-plugin"]
+    }
+  },
+  "require": {
+    "wpackagist-plugin/secure-custom-fields": "6.4.1"
```

Comment:
```suggestion
    "wpackagist-plugin/secure-custom-fields": "^6.4"
```

Otherwise, it will always install that specific version; let's not take for granted that readers are familiar with Composer 🙂